workflow/cases/coupled_free_forecast.yaml

```diff
@@ -34,7 +34,7 @@ case:
     LEVS: 65
     DELTIM: 450
     SEEDLET: 10
-    CCPP_SUITE: FV3_GFS_v15p2_coupled
+    CCPP_SUITE: FV3_GFS_v16beta_coupled
```

Comment:
@JianKuang-NOAA we aren't quite ready to jump to this yet.  can you please leave this as     
CCPP_SUITE: FV3_GFS_v15p2_coupled		
for the time being?

Reply:
OK. I will revert this change.